sdk/tests/deqp/modules/shared/glsFboCompletenessTests.js

```diff
@@ -147,8 +150,8 @@ goog.scope(function() {
          * @type {Array<number>}
          */
         glsFboCompletenessTests.s_extTextureRgHalfFloatTexFormats = [
-            glsFboUtil.formatkey(gl.RED, gl.HALF_FLOAT_OES),
-            glsFboUtil.formatkey(gl.RG, gl.HALF_FLOAT_OES)
+            glsFboUtil.formatkey(gl.RED, gl['HALF_FLOAT_OES']),
```

Comment:
This one might not work because this enum is exposed on the OES_texture_half_float extension object: https://www.khronos.org/registry/webgl/extensions/OES_texture_half_float/ .